Sources/DeepLearning/Operators.swift

```diff
@@ -98,12 +98,14 @@ public extension Tensor where Scalar: BinaryFloatingPoint {
 }
 
 //===------------------------------------------------------------------------------------------===//
-// Convolution and pooling
+// Convolution and Pooling
 //===------------------------------------------------------------------------------------------===//
 
 /// A padding scheme. Used by padding, convolution, and pooling ops.
 // @_frozen // SR-9739
 public enum Padding {
+    /// The "explicit" padding scheme.
```

Comment:
We should document what "explicit" means. Adapting from [TF 2.0 docs](https://www.tensorflow.org/versions/r2.0/api_docs/python/tf/nn/conv2d):

"An array indicating the explicit padding sizes at the start and end of each dimension."